analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -128,22 +132,22 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
 
     private static class DependencyTreeModelBuilder implements ToolingModelBuilder {
         static DependencyImpl dependencyFromDisplayName(String displayName, List<Dependency> dependencies,
-                                                        String error) {
+                                                        String error, String warning) {
             if (displayName.startsWith('project :')) {
                 def coordinates = displayName.split(':', 2)
                 return new DependencyImpl(groupId: '<project>', artifactId: coordinates[1], dependencies: dependencies,
-                        error: error.toString())
+                        error: error, warning: warning)
```

Comment:
The arguments being Strings does not protect from passing GStrings here. We are doing this below, see lines 281, 285, 288. Better also add `toString()` to the warning because it's easy to accidentally pass a GString.

Reply:
Done.